### ISSUE TITLE
数据质量评估模型权重缺失

### ISSUE THREAD

**Issue by Kouyouyi** [alibaba-damo-academy/RynnVLA-002#19]
请问path/pretrained_models/i3d/i3d_torchscript.pt权重在哪里，用来评估数据生成质量的那个脚本里的

**Comment by thz27** [comment]
> 请问path/pretrained_models/i3d/i3d_torchscript.pt权重在哪里，用来评估数据生成质量的那个脚本里的

https://download.pytorch.org/models/vgg16-397923af.pth

**Comment by pumfish** [comment]
https://github.com/alibaba-damo-academy/RynnVLA-002/blob/e548ccc2977ba4fc06013de18192fa6245e454a4/rynnvla-002/exps_libero_world_model/calculate_world_model_performance.py#L18-L31

> > 请问path/pretrained_models/i3d/i3d_torchscript.pt权重在哪里，用来评估数据生成质量的那个脚本里的
> 
> https://download.pytorch.org/models/vgg16-397923af.pth

Hello，I tried using the weights you provided. I noticed that ```lpips.LPIPS(net='vgg')``` attempts to download to the same path as your provided weights. Could you confirm whether the ```I3D model``` weights are correctly supplied?

**Comment by thz27** [comment]
> [RynnVLA-002/rynnvla-002/exps_libero_world_model/calculate_world_model_performance.py](https://github.com/alibaba-damo-academy/RynnVLA-002/blob/e548ccc2977ba4fc06013de18192fa6245e454a4/rynnvla-002/exps_libero_world_model/calculate_world_model_performance.py#L18-L31)
> 
> Lines 18 to 31 in [e548ccc](/alibaba-damo-academy/RynnVLA-002/commit/e548ccc2977ba4fc06013de18192fa6245e454a4)
>  class Evaluator(nn.Module): 
>      def __init__(self, i3d_path='path/pretrained_models/i3d/i3d_torchscript.pt', detector_kwargs=None, max_batchsize=None): 
>          super().__init__() 
>          self.mse = nn.MSELoss(reduction='none') 
>          self.lpips = lpips.LPIPS(net='vgg').cuda().eval() # Ensure lpips is on cuda and eval mode 
>          self.psnr = piqa.PSNR(epsilon=1e-08, value_range=1.0, reduction='none') 
>          self.ssim = piqa.SSIM(window_size=11, sigma=1.5, n_channels=3, reduction='none').cuda() 
>   
>          self.i3d_model = torch.jit.load(i3d_path).cuda().eval() 
>          self.max_batchsize = max_batchsize # Keep max_batchsize if needed for LPIPS batching 
>   
>          # Default detector_kwargs if not provided 
>          self.detector_kwargs = detector_kwargs if detector_kwargs is not None else dict(rescale=True, resize=True, return_features=True) 
>   
> 
> > > 请问path/pretrained_models/i3d/i3d_torchscript.pt权重在哪里，用来评估数据生成质量的那个脚本里的
> > 
> > 
> > https://download.pytorch.org/models/vgg16-397923af.pth
> 
> Hello，I tried using the weights you provided. I noticed that `lpips.LPIPS(net='vgg')` attempts to download to the same path as your provided weights. Could you confirm whether the `I3D model` weights are correctly supplied?

<img width="375" height="498" alt="Image" src="https://github.com/user-attachments/assets/496e6d70-7d7e-4d08-af5e-0e63bed2d38b" />

**Comment by pumfish** [comment]
Thank you for your quick reply. I realized that https://download.pytorch.org/models/vgg16-397923af.pth is actually ```i3d_torchscript.pt``` .btw, I noticed that ```lpips``` and ```i3d``` use the same weights, does this mean they are using the same model?


<img width="1656" height="273" alt="Image" src="https://github.com/user-attachments/assets/a7c126fc-6ca5-4628-910d-e05b075ff8dd" />
